EmbeddedSocialClient/sdk/src/main/java/com/microsoft/embeddedsocial/account/AccountDataStorage.java

```diff
@@ -50,6 +52,31 @@ static void store(Context context, AccountData accountData) {
 			.apply();
 	}
 
+	/**
+	 * Saves the current user's hashed email to the disk.
+	 * @param context runtime context
+	 * @param email hash of email
+	 */
+	static void storeEmail(Context context, String email) {
```

Comment:
consider renaming email to hashedEmail. When I read the comment describing this method, and saw that the input variable is email, I expected this method to do the hashing.

Reply:
Agreed, I thought the same thing and then went digging elsewhere to make sure it was hashed somewhere.